source/content/php-runtime-generation-2.md

```diff
@@ -33,6 +33,8 @@ During the beta phase, you can opt in to test your sites on the new PHP runtime.
 - Identify any potential issues early
 - Provide valuable feedback to our team
 
+We currently recommend only using the PHP Runtime Generation 2 beta for testing with non-Live environments.
```

Comment:
This should probably be a big bold warning at the top of the page, yes?

Reply:
I agree, done 👍